packages/core-suggest/core-suggest.js

```diff
@@ -49,6 +84,25 @@ export default class CoreSuggest extends HTMLElement {
 
   escapeHTML (str) { return escapeHTML(str) }
 
+  _clearLiveRegion () {
+    if (this._ariaLiveTimeout) clearTimeout(this._ariaLiveTimeout) // Clear existing timeout
+    this._ariaLiveSpan.textContent = ''
+  }
+
+  /**
+   * @param {String} label
+   * @returns {void}
+   */
+  _pushToLiveRegion (label) {
```

Comment:
I suspect it may be useful to invoke this, and _clearLiveRegion directly for users. Maybe it ought to be renamed without underscore prefix. Might be just as useful to refactor into a private helper though.